pytorch/buildspec-sm-lite-1-13.yml

```diff
@@ -0,0 +1,116 @@
+account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
```

Comment:
Let's stick to the naming convention we discussed before. I'll rename this file. The buildspec files that include the framework version are meant to exist as references or be used by dedicated pipelines, rather than be the "standard" ones.

Also, we don't want the `buildspec-sm-lite-1-13.yml` buildspec for PT 1.13 to clash with the `buildspec-1-13.yml` buildspec, when the latter is created, which is why we have the name for the sm-lite buildspec created this way.